pyproject.toml

```diff
@@ -13,9 +13,14 @@ rank = "cli.main:app"
 [tool.poetry.dependencies]
 python = "^3.10"
 typer = {extras = ["all"], version = "^0.9.0"}
+pytest = "^7.4.0"
+elasticsearch = "^8.9.0"
+boto3 = "^1.28.16"
+httpx = "^0.24.1"
 
 [tool.poetry.group.dev.dependencies]
 black = {extras = ["d"], version = "^23.7.0"}
+isort = "^5.12.0"
```

Comment:
Think you also need to tell isort not to interfere with black https://pycqa.github.io/isort/docs/configuration/black_compatibility.html